apps/gitness/src/App.tsx

```diff
@@ -145,6 +146,39 @@ export default function App() {
       path: '/',
       element: <RootWrapper />,
       children: [
+        // 🚨 Routes to fix 🚨
+        {
```

Comment:
These links are added for testing and as discussed, these links have to be updated with proper screens.